tests/test_fracOutsideITL.py

```diff
@@ -0,0 +1,64 @@
+"""
+Evaluates the fraction of a sensor surface outside the nominal allowed range, using
+absolute height quantiles
+"""
+from __future__ import print_function
+import os
+import unittest
+from MetrologyData import md_factory, XyzPlane
+import metUtils
+
+class fracOutsideITLTestCase(unittest.TestCase):
+    "TestCase class for frac_outside for ITL absolute height."
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def test_frac_outside(self):
+        """
+        Verifies evaluation of the fraction of a sensor surface outside the
+        allowed range (+/-9 microns from Znom)
+        """
+        infile = os.path.join(os.environ['METROLOGYDATAANALYSISDIR'], 'tests',
+                                         'ITL_vendor_metrology_data.txt')
+        dtype = 'ITL'
+
+        sensorData = md_factory.create(infile, dtype=dtype)
+
+        # Set reference plane at Znom=12.992 mm
+        #
+        sensorData.set_ref_plane(XyzPlane(0, 0, 12992.))
+
+        sensorData.quantile_table()
+
+        frac_outside = metUtils.frac_outside(sensorData.quantiles)
+
+        # Check against previous result 
+        #
+        print('frac_outside:  %s' % frac_outside)
```

Comment:
Unit tests should not print any output unless there is an error or test failure.  If the asserts below do fail, the unittest framework will report the values being compared.

Reply:
I've made this change too in the version I just pushed to the LSSTTD-1152 branch.  Thank you.